runtime/kusama/src/lib.rs

```diff
@@ -1084,11 +1068,28 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPallets,
-	BabeEpochConfigMigrations,
+	(BabeEpochConfigMigrations, KillOffchainPhragmenStorageTest),
 >;
 /// The payload being signed in the transactions.
 pub type SignedPayload = generic::SignedPayload<Call, SignedExtra>;
 
+parameter_types! {
+	pub const StakingPrefix: &'static str = "Staking";
+}
+
+/// This is only for testing. The main migration is inside staking's `on_runtime_upgrade`.
+pub struct KillOffchainPhragmenStorageTest;
```

Comment:
This is just for testing, as the name suggests.

Reply:
does this mean it will be removed before merge?

Reply:
I will remove it later, since I will probably want to test this one last time before enactment. 

The migration is noop, rest assured it won't cause any issues.